### DISCUSSION TITLE
Is this an error and what does it mean?

### DISCUSSION THREAD

**Discussion by dacixn** [KadeArchive/Kade-Engine#484] [Q&A]
![image](https://user-images.githubusercontent.com/77625708/118399058-46a40200-b629-11eb-87a6-52f730757728.png)

**Comment by puyoxyz** [comment]
First off, unrelated to the error, **do not build with sudo**.

Anyways, that's an error with the newgrounds library, you can try to fix it by reinstalling it (`haxelib install newgrounds`)

**Reply by dacixn** [reply]
I'm gonna try again this time without sudo, I also updated the newgrounds library

**Reply by dacixn** [reply]
![image](https://user-images.githubusercontent.com/77625708/118400520-fa0ff500-b62f-11eb-8b68-f0602284500e.png)
it's giving me this error now

**Reply by puyoxyz** [reply]
Make sure you have permissions for the kade engine folder
